plugins/languages/rust/crates.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config {
+    name = "crates";
+    originalName = "crates.nvim";
+    defaultPackage = pkgs.vimPlugins.crates-nvim;
+
+    maintainers = [maintainers.GaetanLepage];
+
+    settingsOptions = {
+      on_attach = helpers.defaultNullOpts.mkLuaFn "function(bufnr) end" ''
```

Comment:
Shouldn't we default this to the __lspOnAttach function? I think we should expose this in the helpers in some way, to help  people reuse the default function when creating their own on attach